bftengine/tests/testViewChange/testViewChange.cpp

```diff
@@ -309,10 +391,9 @@ TEST(testViewchangeSafetyLogic_test, empty_correct_VC_msgs) {
 
 int main(int argc, char** argv) {
   ::testing::InitGoogleTest(&argc, argv);
-  assert(system("../../../tools/GenerateConcordKeys -n 4 -f 1 -o replica_keys_") != -1);
+  Assert(system("../../../tools/GenerateConcordKeys -n 4 -f 1 -o replica_keys_") != -1);
```

Comment:
do we still need this, given that we no longer create a dummy replica?

Reply:
We don't need it anymore, thanks :)